src/Header/_header-settings.scss

```diff
@@ -1,11 +1,11 @@
 @import '~@nice-digital/nds-core/scss/core';
 
 // TODO: Remove this override when the design system is rolled out everywhere
-$colour-body: $colour-true-white;
+$nds-colour-body: $nds-colour-true-white;
 
-$sub-nav-height: $spacing-x-large;
+$sub-nav-height: $nds-spacing-x-large;
 
-$colour-nav-background: $colour-nice-teal;
-$colour-nav-text: $colour-text-inverse;
+$nds-colour-nav-background: $nds-colour-nice-teal;
```

Comment:
`$nds-colour-nav-background` and `$nds-colour-nav-text` are specific to global nav so don't need to be prefixed nds